website/content/en/docs/kubebuilder/quickstart.md

```diff
@@ -285,8 +285,12 @@ $ make run ENABLE_WEBHOOKS=false
 
 #### Build and push the image
 
-Build the operator image and push it to a registry.
+Before building the operator image, ensure the generated Dockerfile references
+the base image you want. You can change the default "runner" image `gcr.io/distroless/static:nonroot`
+by replacing its tag with another, for example `alpine:latest`, and removing
+the `USER: nonroot:nonroot` directive.
 
+To build and push the operator image, use the following `make` commands.
```

Comment:
we should suggest uss the operator-sdk commands instead of the make ones.

Reply:
This isn't really a change to the documentation content, just grammar. However `operator-sdk build` was only ported over for backwards compatibility with existing scripts on new project layouts. Users should be switching to `make docker-build`.